cdap-ui/app/directives/log-viewer-preview/log-viewer-preview.html

```diff
@@ -43,6 +43,13 @@
     </div>
     <div class="log-actions">
       <div class="log-action-container">
+        <span class="include-system">
+          <i ng-class="{'fa fa-square': !LogViewerPreview.includeSystemLogs,
+            'fa-check-square': LogViewerPreview.includeSystemLogs}"
```

Comment:
you are missing the `fa` class in here. I would add a class to the `<i>` `class="fa"` and just do the `ngClass` for the `fa-square` & `fa-check-square`